tests/container

```diff
@@ -147,5 +147,14 @@ lxc exec u1 -- mkdir /root/sys
 lxc exec u1 -- mount -t sysfs sysfs /root/sys
 lxc delete -f u1
 
+echo "==> Test exit codes when container disconnects during lxc exec"
+lxc launch "${IMAGE}" c1
+
+(sleep 1 && lxc stop -f c1) &
+lxc exec c1 -- sleep 10 || exitCode=$?
+[ "${exitCode:-0}" -eq 129 ]
```

Comment:
Hey, I was about to comment that checking for `SIGKILL` was potentially risky as I'd expect `SIGTERM` to also be a possible return code, no?

My assumption is that `lxc stop` sends a `SIGTERM` first, wait a bit then send a `SIGTERM`. Assuming that assumption is valid, would it be possible the first `SIGTERM` is enough to stop everything cleanly in the instance and thus change the return code?

Also, I'd probably want that `128+SIG_NUMBER_` logic briefly mentioned here cause `SIGKILL=9` and `SIGTERM=15` is widely known but `129` or `137` are more like magic numbers to me ;)

Reply:
Sure! But could you please elaborate on the "And what about a graceful stop not aborting anything" case?

Reply:
> But could you please elaborate on the "And what about a graceful stop not aborting anything" case?

Just a plain, `lxc launch`, wait ready then `lxc stop`.